packages/genesys-spark-chart-components/package.json

```diff
@@ -49,7 +49,7 @@
   "dependencies": {
     "@redsift/d3-rs-intl": "^0.4.0",
     "vega": "5.31.0",
-    "vega-embed": "6.29.0",
+    "vega-embed": "^7.0.2",
```

Comment:
Remove the caret. We have run into issues where even patch changes in vega modules have broken stuff.

Reply:
Removed, looking into the build failure atm looks like they changed something with imports from version 7 onwards may need to tweak the tsconfig in charts package potentially.